spec/unit/calendar_spec.rb

```diff
@@ -27,19 +27,19 @@
     before do
       @cal = Calendar.new("a-calendar", "title" => "UK bank holidays",
                                         "divisions" => {
-          "kablooie" => {
-            "2012" => [1],
-            "2013" => [3],
-          },
-          "fooey" => {
-            "2012" => [1, 2],
-            "2013" => [3, 4],
-          },
-          "gooey" => {
-            "2012" => [2],
-            "2013" => [4],
-          },
-        })
+                                          "kablooie" => {
+                                            "2012" => [1],
+                                            "2013" => [3],
+                                          },
+                                          "fooey" => {
+                                            "2012" => [1, 2],
+                                            "2013" => [3, 4],
+                                          },
+                                          "gooey" => {
+                                            "2012" => [2],
+                                            "2013" => [4],
+                                          },
+                                        })
     end
```

Comment:
This indentation looks rather odd. How about this?
```ruby
before do
  @cal = Calendar.new(
    "a-calendar",
    "title" => "UK bank holidays",
    "divisions" => {
      "kablooie" => {
        "2012" => [1],
        "2013" => [3],
      },
      "fooey" => {
        "2012" => [1, 2],
        "2013" => [3, 4],
      },
      "gooey" => {
        "2012" => [2],
        "2013" => [4],
      },
    }
  )
end
```

Reply:
Sure, updated 👍